codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeSpecUtils.java

```diff
@@ -191,6 +195,26 @@ public List<AuthType> allServiceAuthTypes() {
         return result.stream().sorted().collect(Collectors.toList());
     }
 
+    public Set<Class<?>> allServiceConcreteAuthSchemeClasses() {
```

Comment:
This replaces the use of `allServiceAuthTypes`. So I think remove that method. If you wanted to keep it, maybe it's a private method that this method calls and then processes it. I don't like it being left there as a public method, because it seems generically useful, but has sorting behavior and use of List as opposed to Set, which was specifically for how it was being used for codegen'ing the authSchemes Map. In short, I think delete `allServiceAuthTypes`.

Reply:
Removed.